kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/superuser-credentials-form/index.vue

```diff
@@ -0,0 +1,197 @@
+<template>
```

Comment:
@christianmemije look over validation please?  A bit different than your approach

Reply:
Left a few comments, looks good.

Reply:
Yeah. Includes stops after finding it the first time, so I don't think it'll ever have consequences. I'll look into some solutions, though